examples/table-syntax/table_syntax.bal

```diff
@@ -17,11 +17,10 @@ function increaseSalary(int n) {
     foreach Employee e in t {
         e.salary += n;
     }
-
```

Comment:
I think this newline was intentional. I guess we mark the code segment related to the comment above, using this newline.  Without it both `}`  get captured under the comment.